test/example/type-builder-nominal.cpp

```diff
@@ -433,6 +435,26 @@ void test_subtypes() {
       built = *builder.build();
     }
     assert(LUB(built[1], built[0]) == built[0]);
+
+    // Test SubTypes utility code, which also verifies the above. Create a tiny
+    // module that uses the types in the locals of a function, so the types are
+    // used.
```

Comment:
I would strongly prefer adding this as a gtest. test/gtest/type-builder.cpp already tests many other type-adjacent utilities like this, so this could be added to that file.

Reply:
Ok, moved to gtest.